arangod/Aql/ExecutionBlockImpl.cpp

```diff
@@ -1311,6 +1318,9 @@ ExecutionBlockImpl<Executor>::executeWithoutTrace(AqlCallStack stack) {
 
   LOG_QUERY("007ac", DEBUG) << "starting statemachine of executor " << printBlockInfo();
   while (_execState != ExecState::DONE) {
+    // We can never keep state in the skipCounter
+    TRI_ASSERT(clientCall.getSkipCount() == 0);
+    auto oldState = _execState;
```

Comment:
Please remove this line, as `oldState` is an unneeded variable and will cause a compile warning (or error, if warnings are propagated to errors).
```suggestion
```

Reply:
Thanks for spotting, left-over of debugging ;(